src/dialogs/voice-command-dialog/ha-voice-command-dialog.ts

```diff
@@ -16,7 +16,13 @@ import {
 } from "lit-element";
 import { HomeAssistant } from "../../types";
 import { fireEvent } from "../../common/dom/fire_event";
-import { processText } from "../../data/conversation";
+import {
+  processText,
+  getAgentInfo,
+  setConversationOnboarding,
+  AgentInfo,
+  SpeechRecognition,
```

Comment:
I don't think this should be in data. It ha nothing to do with Home Assistant. Wouldn't it be better placed in `util/` ?